src/addresses.h

```diff
@@ -374,6 +374,8 @@
 #define RCT2_ADDRESS_NEXT_WEATHER_GLOOM				0x013CA751
 #define RCT2_ADDRESS_CURRENT_RAIN_LEVEL				0x013CA752
 #define RCT2_ADDRESS_NEXT_RAIN_LEVEL				0x013CA753
+#define RCT2_ADDRESS_CURRENT_SNOW_LEVEL				0x013CA755
+#define RCT2_ADDRESS_NEXT_SNOW_LEVEL				0x013CA756
```

Comment:
You can't do this, you are writing over new items. Just define these like normal C variables.

This doesn't mean you can not save them to saved games. The only way of overcoming that is using rain level.